docs/antrea-l7-network-policy.md

```diff
@@ -201,6 +203,91 @@ spec:
         - http: {}       # automatically dropped, and subsequent rules will not be considered.
 ```
 
+### TLS
+
+A typical layer 7 NetworkPolicy TLS protocol is as below:
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: NetworkPolicy
+metadata:
+  name: ingress-allow-tls-request-to-api-v2
```

Comment:
The name doesn't reflect what the policy really does

Reply:
Updated to `ingress-allow-tls-hanshake`